packages/itwin/measure-tools/src/measurements/DistanceMeasurement.ts

```diff
@@ -474,9 +503,9 @@ export class DistanceMeasurement extends Measurement {
         QuantityType.LengthEngineering
       );
 
-    const distance = this._startPoint.distance(this._endPoint);
-    const run = this._startPoint.distanceXY(this._endPoint);
-    const rise = this._endPoint.z - this._startPoint.z;
+    const distance = this.sheetToWorldScale * this._startPoint.distance(this._endPoint);
+    const run = this.sheetToWorldScale * Math.abs(this._endPoint.x - this._startPoint.x);
+    const rise = this.sheetToWorldScale * Math.abs(this._endPoint.y - this._startPoint.y);
```

Comment:
Rise can be positive or negative, no need for `Math.abs`.
(Run is always positive since it's a distance, which is correct)